packages/ts/react-form/src/index.ts

```diff
@@ -72,6 +73,13 @@ export type UseFormResult<M extends AbstractModel> = Omit<UseFormPartResult<M>,
     update(): void;
   }>;
 
+export type UseFormArrayPartResult<M extends ArrayModel, M1 extends AbstractModel = AbstractModel> = Omit<
+  UseFormPartResult<M>,
+  'field'
+> & {
+  items: readonly M1[];
```

Comment:
We have an `ArrayItemModel<M>` type that extracts the type from `ArrayModel`:
```suggestion
  items: ReadonlyArray<ArrayItemModel<M>>;
```